### DISCUSSION TITLE
Working with Litellm

### DISCUSSION THREAD

**Discussion by thedunston** [SecureAI-Tools/SecureAI-Tools#55] [Show and tell]
Post Deleted. I apologize for posting in the wrong forum. Very sorry.

**Comment by JayNakrani** [comment]
Sorry, I do not understand. Did you mean you got SecureAI Tools working with LiteLLM?

**Reply by thedunston** [reply]
Yes, that is correct.

**Reply by ishaan-jaff** [reply]
Hi @thedunston - I'm the litellm maintainer, just found this. Do you use the LiteLLM Proxy ?

**Comment by thedunston** [comment]
Oh geez! I posted in the wrong GitHub. That is embarrassing and I apologize.
